bot.user.js

```diff
@@ -511,18 +566,29 @@ var bot = (function() {
                 if (bot.tickCounter > 15) {
                     bot.tickCounter = 0;
 
-                    window.sortedFood = bot.getSortedFood();
-                    window.currentFood = window.sortedFood[0];
+                    var accelerationClusterSize;
+                    if (bot.foodClusteringMode == 1) {
+                        bot.computeFoodGoal();
+                        accelerationClusterSize = 120;
+                    } else {
+                        var foods = bot.getFood();
+                        window.sortedFood = foods.map(bot.foodClusters).sort(bot.sortFoodClusters);
```

Comment:
I assume Hound means the .sort of a .map of foods, but it looks good to me though.